### PR TITLE
new loader document

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@nolouch @iamxy PTAL

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
@tiancaiamao @cwen0 PTAL

File: tools/loader.md [hunk 1]
```diff
@@ -16,11 +16,15 @@
...
+* 支持表级别的并发导入，分散写入热点
+
+* 支持对单个大表并发导入，分散写入热点
+
 * 支持 mydumper 数据格式
 
 * 出错重试
 
-* 断点续导
+* 断点续导
```
Comment (tiancaiamao) [inline tools/loader.md]: remove the useless whitespace

**Comment by zhangjinpeng1987** [comment]
PTAL

**Comment by cwen0** [comment]
LGTM

**Comment by nolouch** [comment]
LGTM

**Pull request merged**